packages/elements/tailwind.config.js

```diff
@@ -17,6 +17,9 @@ module.exports = {
   // Custom selector for dark theme
   darkMode: ['class', ':host([theme="dark"])'],
   content: ['./src/**/*.ts'],
+  plugins: [
+    require('@tailwindcss/forms'),
```

Comment:
What is this used for?

Reply:
Correctly showing the checkbox styling. Without it, the styling breaks.